GPU/Common/ShaderUniforms.h

```diff
@@ -29,21 +30,21 @@ struct UB_VS_FS_Base {
 	float uvScaleOffset[4];
 	float depthRange[4];
 	// Rotation is used only for software transform.
-	float fogCoef[2]; float stencil; float rotation;
 	float matAmbient[4];
 	float cullRangeMin[4];
 	float cullRangeMax[4];
 	uint32_t spline_counts; uint32_t depal_mask_shift_off_fmt;  // 4 params packed into one.
 	uint32_t colorWriteMask; float mipBias;
 	// Fragment data
 	float fogColor[4];     // .w is unused
-	float texEnvColor[4];  // .w is unused
+	float texEnvColor[3];
+	uint32_t colorTestMask;
 	int alphaColorRef[4];
-	int colorTestMask[4];
-	float blendFixA[4];  // .w is unused
-	float blendFixB[4];  // .w is unused
+	float blendFixA[3]; float stencil;
+	float blendFixB[3]; float rotation;
 	float texClamp[4];
-	float texClampOffset[4];  // .zw are unused
+	float texClampOffset[2];  // .zw are unused
```

Comment:
I guess this comment is outdated with fogCoef, which should maybe be on the same line for clarity?

-[Unknown]

Reply:
Indeed